packages/firebase_ml_vision/ios/Classes/CloudLabelDetector.m

```diff
@@ -32,20 +35,4 @@ + (void)handleDetection:(FIRVisionImage *)image
            result(labelData);
          }];
 }
-
-+ (FIRVisionCloudDetectorOptions *)parseOptions:(NSDictionary *)optionsData {
```

Comment:
I moved this method to `FirebaseMLVisionPlugin.m` so it can be used by all cloud detectors.